readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -374,48 +382,65 @@ def __init__(self, *args, **kwargs):
 
 class LocalMediaBuilder(BaseSphinx):
     sphinx_builder = 'readthedocssinglehtmllocalmedia'
-    sphinx_build_dir = "_readthedocs/htmlzip"
+    relative_output_dir = "_readthedocs/htmlzip"
 
     def _post_build(self):
         """Internal post build to create the ZIP file from the HTML output."""
-        sphinx_build_dir = os.path.join(self.project_path, self.sphinx_build_dir)
-        temp_zip_file = f"/tmp/{self.project.slug}-{self.version.slug}.zip"
-        target_file = os.path.join(
-            sphinx_build_dir,
-            f"{self.project.slug}.zip",
+        target_file = os.path.abspath(
+            os.path.join(
+                self.absolute_output_dir,
+                # TODO: shouldn't this name include the name of the version as well?
+                # It seems we were using the project's slug previously.
+                # So, keeping it like that for now until we decide make that adjustment.
+                f"{self.project.slug}.zip",
+            )
         )
 
-        # FIXME: ``arcname`` is invalid with this command still, but it's close
+        # Move the directory into a temporal directory,
```

Comment:
This is the security PR fix just getting pulled in, right?

Reply:
```suggestion
        # **SECURITY CRITICAL: Advisory GHSA-hqwg-gjqw-h5wg**
        # Move the directory into a temporal directory,
```

Reply:
Yes.